### ISSUE TITLE
Docs for CSV 1.2.1 render a 404

### ISSUE THREAD

**Issue by beatrichartz** [hexpm/hex_web#179]
When I upload the docs for `CSV` version 1.2.1, I get:
```
Docs successfully generated.
View them at "doc/index.html".
[#########################] 100%
Published docs for csv v1.2.1
Hosted at http://hexdocs.pm/csv/1.2.1/
```
However, http://hexdocs.pm/csv/1.2.1/ is rendering a 404 - is there something wrong with these docs? When I open them locally under `doc/index.html`, they work fine.

**Comment by josevalim** [comment]
Can you try publishing them? Did you receive an email saying if they were
published with success or fail?

-- 


*José Valim*
www.plataformatec.com.br
Skype: jv.ptec
Founder and Director of R&D

**Comment by josevalim** [comment]
Oops, Can you try *republishing* them? :)

-- 


*José Valim*
www.plataformatec.com.br
Skype: jv.ptec
Founder and Director of R&D

**Comment by beatrichartz** [comment]
Republished them again right now using `mix hex.docs` - [Here's a copy of the folder it uploaded](https://dl.dropboxusercontent.com/u/2323078/doc.zip).

I received an email when the docs for 1.2.1 were originally published on October 17, but did not visit the site then. I did not receive an email when I published an hour ago and now.

It seems to me like a broken version of the documentation is present and does not get replaced by the one I'm republishing. The version present on the server is generated with `ex_doc` `0.10.0` and redirects to `overview.html`, which does not exist. I had to revert `ex_doc` to `0.9` to get the documentation working locally, and that's the one I uploaded today.
